package.json

```diff
@@ -129,7 +129,7 @@
     "audit-ci": "^7.1.0",
     "chokidar": "^3.6.0",
     "concurrently": "^9.1.2",
-    "cypress": "^14.0.0",
+    "cypress": "^14.0.2",
```

Comment:
Cypress is completely broken on my laptop so I haven't added any integration tests - will come back to these when Cypress is fixed.